crates/cheatcodes/src/defs/mod.rs

```diff
@@ -0,0 +1,181 @@
+//! Cheatcode definitions.
+
+mod hevm;
+pub use hevm::HEVM;
+
+use alloy_sol_types::SolCall;
+use serde::{Deserialize, Serialize};
+
+/// Stable specification of a single cheatcode.
+#[derive(Serialize, Deserialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+#[non_exhaustive]
+pub struct Cheatcode<'a> {
+    // Automatically-generated fields.
+    /// The cheatcode's identifier. Corresponds to the function's name,
+    /// thus is not unique if the function is overloaded.
+    pub id: &'a str,
```

Comment:
Why not just use one of the signature fields, or equivalently the selector, as the ID so the ID can be unique? Or if there's no benefit to uniqueness, maybe we can remove this field?

Reply:
Changed to use the ID generated by the macro, which is `name`, or `<name>_<index>` when overloaded
Can remove, but I don't think it hurts